library/src/main/java/org/xmtp/android/library/Group.kt

```diff
@@ -153,20 +153,21 @@ class Group(val client: Client, private val libXMTPGroup: FfiGroup) {
         return libXMTPGroup.isActive()
     }
 
-    fun addedByAddress(): String {
-        return libXMTPGroup.addedByAddress()
+    fun addedByInboxId(): String {
+        return libXMTPGroup.addedByInboxId()
     }
 
     fun permissionLevel(): GroupPermissions {
-        return metadata.policyType()
+        TODO()
+//        return metadata.policyType()
```

Comment:
@cameronvoell I assume you'll be updating this with your new permissions work so just commenting out as a TODO for now.

Reply:
yep, I can fix that in my permissions PR 👍